app/assets/stylesheets/era_custom.scss

```diff
@@ -16,3 +16,42 @@ div.terms-of-use-custom {
   margin-left: 30px;
   margin-right: 30px;
 }
+
+#content .info-pane p > a {
+  text-decoration: none;
+  color: #337ab7 !important;
```

Comment:
Make sass variable for color code and use that.
Also should never use `!important` if you can help it (and considering your referencing an ID and lots of classes/html tags here, you shouldn't need it?).